<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraWindow</class>
 <widget class="QWidget" name="CameraWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1649</width>
    <height>792</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame_CamOperate">
   <property name="geometry">
    <rect>
     <x>1340</x>
     <y>90</y>
     <width>131</width>
     <height>361</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_ParmsSet">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>620</y>
     <width>241</width>
     <height>171</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_ExplosureTime">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>37</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>曝光时间</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_FpsSet">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>77</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>增益设置</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ExplosureTime">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_FpsSet">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ParamsSet">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>参数设置</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_WinInfo">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>560</y>
     <width>271</width>
     <height>191</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_CamIndex">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>相机参数</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_FileName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>54</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>文件名字</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_FilePath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>99</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>文件路径</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_NumOfCam">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>143</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>相机数目</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_CamIndex">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_FileName">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_FilePath">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_NumOfCam">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_SampleImg" native="true">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>230</y>
     <width>491</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_SampleImg">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="DrawWidget" name="widget_CurInput" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>471</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_CurInput">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_CloseCam">
   <property name="geometry">
    <rect>
     <x>1500</x>
     <y>200</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭相机</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_GetImg">
   <property name="geometry">
    <rect>
     <x>1500</x>
     <y>350</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>获取图像</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ListDevice">
   <property name="geometry">
    <rect>
     <x>1500</x>
     <y>120</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>列出设备</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_OpenCam">
   <property name="geometry">
    <rect>
     <x>1500</x>
     <y>160</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>打开相机</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_StartGrab">
   <property name="geometry">
    <rect>
     <x>1500</x>
     <y>250</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>开始采集</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_StopGrab">
   <property name="geometry">
    <rect>
     <x>1510</x>
     <y>290</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>停止采集</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DrawWidget</class>
   <extends>QWidget</extends>
   <header>drawwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
